endpoints.php

```diff
@@ -6,6 +6,71 @@
 
 require __DIR__ . '/controller/class.tapestry-permissions.php';
 require __DIR__ . '/controller/class.tapestry-controller.php';
+require __DIR__ . '/controller/class.user-controller.php';
+
+ // User endpoints
+
+// Get user progress on a tapestry page by post id. 
+// Example: /wp-json/tapestry-tool/v1/users/progress?post_id=44
+add_action( 'rest_api_init', function () {
```

Comment:
@kevinthewebguy @wynnset  Shall we follow the pattern of creating an endpoint like other endpoints we have created here? Just for maintenance and consistency purposes.